apps/web/src/components/organisms/rateLecture/RateLecture.tsx

```diff
@@ -0,0 +1,53 @@
+'use client'
+
+import react from 'react';
+import styles from "./RateLecture.module.scss"
+import { SpeakerCard, SpeakerCardProps} from '@/components/molecules/lectureCard/SpeakerCard/SpeakerCard';
+import { Text } from "@/components/atoms/text/Text";
+import RateStars from '@/components/molecules/rateStars/RateStars';
+import TextArea from '@/components/molecules/textArea/TextArea';
+import Button from "@/components/atoms/button/Button";
+
+
+export interface RateLectureProps{
+    title: string;
+    speakers: SpeakerCardProps[];
+}
+
+const RateLecture: React.FC<RateLectureProps> = ({
+    title,
+    speakers,
+  }) => {
+
+    return (
+        <div className={styles.root}>
+            <div className={styles.rateLecture}>
+                <div className={styles.vertical}>
+                    <Text variant="headS" bold>
+                        {title}
+                    </Text>
+                    <div className={styles.horizontal}>
+                        {speakers.map((speaker, index) => (
+                            <SpeakerCard key={index} {...speaker} />
```

Comment:
W react'cie nie powinno się korzystać z indexów do key'ów, tu z reguły są tylko dwa elementy, ale warto to już sobie zapamiętać i korzystać na przyszłość 😄 

https://react.dev/learn/rendering-lists#why-does-react-need-keys